### DISCUSSION TITLE
Internal server error on deploy (updating dependencies)

### DISCUSSION THREAD

**Discussion by plutonium-239** [deta/cloud-docs#621] [Help]
I have a very simple requiremenets.txt:
```
dash
dash-bootstrap-components
```
(and the package size of `dash` is ~39 MB, and for the other package is ~1 MB)
This is well within the 250 MB limit, so why is it giving an error?

**Comment by etrotta** [comment]
It looks like even including all dependencies from dash, it really doesn't gets anywhere near 250 (perhaps unless you are also including pandas, idk) - but when I tested pip installing these into a fresh venv locally, it seems to get stuck doing something for quite a long amount of time?

I'm not sure what, why, how, or even if it's Dash fault or some dependency, but it sure took a long time - so my guess would be just that deta is timing it out during that time.

**Reply by plutonium-239** [reply]
Yes, I can replicate the same.
Actually, I figured out that `plotly` was taking the most time to install (and is also the biggest package - 91 MB).
It is a dependency of `dash`.

I tried `pip freeze req.txt`, editing `plotly` out of this req.txt (its not required if you're not making graphs), and then in a new venv, doing `pip install --no-deps -r req.txt`
This did not remain 'stuck' (in installation) like the one with `plotly`.

However, I don't know of any option to tell `deta deploy` to use `--no-dependencies` option in pip install.

**Reply by plutonium-239** [reply]
@etrotta do you know of any?
I would really like if they added this as a simple option (maybe with deploy cli)
